### DISCUSSION TITLE
Suddenly can't acces immich webserver/ip

### DISCUSSION THREAD

**Discussion by VegArchie** [immich-app/immich#13568] [Q&A]
At some point in the last week, my immich server became unreachable. Trying to access immich at its web address results in no page. App does not connect either.

I updated it with the docker pull command today but no change. I can see that the database folders are updating by viewing the modification time in the folder. I am running it on Linux. I have tried rebooting the PC, but nothing changes. 

There is another page for another docker service that I run and can be connected to correctly. I checked the firewall and its ok and correct. Tried localhost and numerical ip. Nothing connects.

Anything anyone can recommended to try?

**Comment by kdpuvvadi** [comment]
Happend same to my instance. Seems like they made change of immich default port to `2283` from `3001` ( commit 465f4639da32144b9eedc4a89c3df761420350d1 with #13185 ). I was pulling my hair and looked at the logs of `immich-server` container, instead of showing `Immich Server is listening on http://[::1]:3001`, it was showing `....:2283`. 

Change docker compose accordingly

```compose.yaml
name: immich

services:
  immich-server:
    .
    .
    ports:
      - 2283:2283
```

**Reply by Benny-bull** [reply]
Thanks, this helped :-)

**Reply by wedsa5** [reply]
i also pulled my hair out thinking it was something wrong with my caddy reverse proxy. i also had `- 2283:3001`. Changing that fixed it for me as well.

**Reply by sonnguyen9800** [reply]
Thank you so much much! Pulling my hair for a while until I saw this thread! Please be healthy and happy with people you love!

**Reply by Am0rphous** [reply]
OMG! Been pulling my hair out! Thank you!!!

**Reply by gtianp** [reply]
It works like a charm. Thank you!!!
